taskcluster/fenix_taskgraph/transforms/push_apk.py

```diff
@@ -40,11 +40,7 @@ def build_worker_definition(config, tasks):
         )
 
         build_type = task["attributes"]["build-type"]
-        # Fenix production doesn't follow the rule {product}-{channel}
-        if build_type == "production":
-            worker_definition["certificate-alias"] = "fenix"
-        # Neither do Fennec flavored builds
-        elif build_type.startswith("fennec-"):
+        if build_type == "beta" or build_type == "nightly":
             worker_definition["certificate-alias"] = build_type
```

Comment:
This is now a different value for `build-type` and I do not know what `certificate-alias` is used for.

Reply:
Let's keep the same values for certificate alias, for now. We have to coordinate the renaming between here and there https://github.com/mozilla-releng/scriptworker-scripts/blob/d9f5f854fd3aff0db498ad6e502e00f63d79f6fe/pushapkscript/docker.d/worker.yml#L79. I'll take care of it in a followup.

In order words, we could either change the logic here to keep the output the same, or we could put all values in kind.yml. The latter is probably simpler and would look like this: 

```yaml
 job-template: 
    worker:
        certificate-alias:
            # TODO rename these values to use non-legacy terms.
            by-build-type:
                nightly: fenix-production
                beta: fennec-beta
                release: fennec-production
```

You'll have to update https://github.com/mozilla-mobile/fenix/blob/e2e684b4d452378cf07c70b00f921afe9b5b687a/taskcluster/fenix_taskgraph/transforms/push_apk.py#L21 to include `worker.certificate-alias`.